eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -140,7 +140,17 @@ public boolean hasRho() {
 
     @Override
     public void put(final int pos, final Phi object) {
-        this.put(this.attr(pos), object);
+        final String name = this.attr(pos);
+        final AtWithRho attribute = (AtWithRho) this.attrs.get(name);
+        if (!(attribute.getOrigin() instanceof AtVoid)) {
+            throw new ExReadOnly(
+                String.format(
+                    "There are no void attributes left, can't set the %s one",
```

Comment:
@Suban05 the message is not quite correct. There are still may be void attributes ready to be set, but we've just missed it by providing wrong index. So let's say something like: `Can't put attribute with position %d because it's not void one`